libraries/google_organization.rb

```diff
@@ -17,10 +17,11 @@ class GoogleOrganization < GcpResourceBase
     "
     def initialize(opts = {})
       super(opts)
-      @display_name = opts[:name] || opts[:display_name]
-      catch_gcp_errors do
-        @organization = @gcp.gcp_project_client.get_organization(opts[:name])
-        create_resource_methods(@organization)
+      if opts[:name]
```

Comment:
Thanks for this @lhasadreams - this `if` statement is slightly problematic because the resource will silently do nothing if the name is not provided.  

Could you replace this with something like the following:
```
raise Inspec::Exceptions::ResourceFailed, "google_organization is missing mandatory property 'name'" if opts[:name].nil?
```

Reply:
Sure, that makes a lot of sense - code updated